tensorflow/core/common_runtime/mkl_cpu_allocator.h

```diff
@@ -48,6 +50,120 @@ class MklSubAllocator : public SubAllocator {
   void Free(void* ptr, size_t num_bytes) override { port::AlignedFree(ptr); }
 };
 
+/// CPU allocator that handles small-size allocations by calling
```

Comment:
Sorry for nitpicking, but our formatting style only uses two slashes `//` for the comment. Could you please change all the comment lines beginning with `///` to just `//`? Thank you very much!